src/bindings/python/.pylintrc

```diff
@@ -44,7 +44,7 @@ symbols=no
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=missing-docstring, too-few-public-methods, too-many-arguments, star-args, arguments-differ, no-member, import-error, no-name-in-module, useless-object-inheritance, fixme
+disable=missing-docstring, too-few-public-methods, too-many-arguments, star-args, arguments-differ, no-member, import-error, no-name-in-module, useless-object-inheritance, fixme, C0330
```

Comment:
Do you happen to know what the "human-readable" code for `C0330` is?  If so, can we use that instead?  Based on [this issue](https://github.com/PyCQA/pylint/issues/289), I believe it is `bad-continuation`.

Reply:
I'll try that, good point.